src/pipes/db-lookup.pipe.ts

```diff
@@ -18,7 +25,21 @@ export function DbLookup(
       target: object.constructor,
       propertyName: propertyName,
       options: validationOptions,
-      constraints: [{ ignoreEmpty, type, findOption }],
+      constraints: [
+        typeof findOption === "object" && !Array.isArray(findOption)
```

Comment:
It doesn't look like any type in the `findOption` union type is an Array, you could remove that check.

Reply:
``` markdown
Agreed & done. 
Removed the unnecessary <!Array.isArray(findOption)>